### ISSUE TITLE
Missing Required global variables

### ISSUE THREAD

**Issue by geographika** [compassinformatics/cpsi-mapview#56]
When loading the application in Chrome the following errors are reported:

```
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "Jsonix" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "SLD_1_0_0" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "Filter_1_0_0" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "GML_2_1_2" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "XLink_1_0" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "Jsonix" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "SLD_1_0_0" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "Filter_1_0_0" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "GML_2_1_2" not found. Are Jsonix needed mappings loaded?
ext-all-debug.js?_dc=1542719471968:9389 [E] Required global variable "XLink_1_0" not found. Are Jsonix needed mappings loaded?
```

Can these be ignored or is there a reference missing?

**Comment by chrismayer** [comment]
I think this is not a problem at the moment. I guess it comes from BasiGX. Maybe @marcjansen or @ahennr know more about this.

This is not perfect to have these warnings and we should handle it somehow (if possible) in the future  but since there are no technical problems I would suggest to postpone.

**Comment by ahennr** [comment]
These are dependencies of the WFS-util in BasiGX. However, the uses methods in cpsi-mapview does not depend on these globals.

**Comment by geographika** [comment]
Resolved with #130
